packages/openapi-ts/src/compiler/index.ts

```diff
@@ -15,9 +15,9 @@ export class TypeScriptFile {
     private _headers: Array<string> = [];
     private _imports: Array<ts.Node> = [];
     private _items: Array<ts.Node | string> = [];
-    private _path: string;
+    private _path: PathLike;
 
-    public constructor(path: string, header: boolean = true) {
+    public constructor(path: PathLike, header: boolean = true) {
```

Comment:
NOTE: we do not need to remove single files as `writeFileSync` will overwrite if already present.